src/Components/Patient/DailyRounds.tsx

```diff
@@ -958,20 +937,8 @@ export const DailyRounds = (props: any) => {
               )}
 
               <div className="mt-4 flex flex-col md:flex-row gap-2 justify-between">
-                <Link
-                  className="btn btn-default bg-white mt-2"
-                  href={`/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}`}
-                >
-                  Back
-                </Link>
-                <button
-                  type="submit"
-                  className="btn btn-primary ml-auto text-base w-full md:w-auto"
-                  onClick={(e) => handleSubmit(e)}
-                >
-                  <i className="fa-regular fa-circle-check mr-2"></i>{" "}
-                  {buttonText}
-                </button>
+                <Cancel onClick={() => goBack()} label="Back" />
```

Comment:
A Back labelled button should not have cancel icon.
Either:
- Remove the label prop so that it's auto labelled as "Cancel" or
- Override the children to have back icon and back label.

Reply:
For consistency, I would choose the first.